src/physics/force_density.jl

```diff
@@ -20,8 +20,56 @@ function _calc_force_density!(storage::AbstractStorage, system::AbstractSystem,
                               params::AbstractPointParameters, each_point_idx)
     storage.b_int .= 0
     storage.n_active_bonds .= 0
-    for point_id in each_point_idx
+    Threads.@threads :static for point_id in each_point_idx
```

Comment:
I just wanted to ignore the chunks and just have one chunk, over which I run a threaded loop. The performance is probably similar, I just wanted to have more threads and therefore less cache per thread, and this was easier than looking at the chunks stuff.